package.json

```diff
@@ -142,19 +145,26 @@
     "stylelint": "9.10.1",
     "tap-colorize": "1.2.0",
     "tape": "4.8.0",
-    "xo": "0.18.2"
+    "typescript": "^3.5.2",
+    "xo": "^0.24.0"
   },
   "xo": {
```

Comment:
@priyank-p, @vsvipul do we have an option to put the `xo` config in a separate file since it's getting longer.

Reply:
No, `xo` doesn't provide a way to use a config file. Though we can use a `.eslintrc` I think.